app/src/test/java/ua/test/mega/tester/junit/AccountProcessorTest.java

```diff
@@ -0,0 +1,127 @@
+package ua.test.mega.tester.junit;
+
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.security.test.context.support.WithAnonymousUser;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.context.junit4.SpringRunner;
+import ua.test.mega.tester.adapters.AccountAdapterInMemory;
+import ua.test.mega.tester.adapters.LoggedInUserAdapterForSpringSecurity;
+import ua.test.mega.tester.adapters.UserAdapterInMemory;
+import ua.test.mega.tester.core.AccountProcessor;
+import ua.test.mega.tester.core.api.AccountAdapter;
+import ua.test.mega.tester.core.api.LoggedInUserAdapter;
+import ua.test.mega.tester.core.api.UserAdapter;
+import ua.test.mega.tester.core.api.model.Account;
+
+import java.math.BigDecimal;
+
+
+@RunWith(SpringRunner.class)
```

Comment:
What is reason to use SpringRunner in this class?

Reply:
SpringRunner provides loading ApplicationContext, so we can implement Junit and integration tests.